app/views/products/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Edit Product</h1>
+
+<section>
+  <%= form_for @product do |f| %>
+    <fieldset>
+      <legend>Edit Market</legend>
+      <%= f.label :name %>
+      <%= f.text_field :name %><br>
+      <%= f.hidden_field :product_id, :value => params[:id] %>
```

Comment:
I don't think we need to use this hidden input field, as the path for our `products#update` action already includes the id (it's something like `/products/ID_HERE`), and in fact our `update` method in `ProductsController` does not use the `product_id` param value at all.